pixel-saver@deadalnix.me/buttons.js

```diff
@@ -47,8 +47,13 @@ function createButtons() {
 	
 	let orders = order.replace(/ /g, '').split(':');
 	
-	orders[0] = orders[0].split(',');
-	orders[1] = orders[1].split(',');
+	// Checking the left button orders, make an empty list when it's not exists
+	if(typeof orders[0] != 'undefined') orders[0] = orders[0].split(',');
+	else orders[0] = [];
+	
+	// Checking the right button orders, make an empty list when it's not exists
+	if(typeof orders[1] != 'undefined') orders[1] = orders[1].split(',');
+	else orders[1] = [];
```

Comment:
You can split an empty string, so there is no need for a else. Just set orders[1] to an empty string is it is not defined, and then resume regular operations.